andela/travel_tool_back
# #160595712 Add isEdit field to the comments table

#### What does this PR do?
This PR adds a field to the comments table to check if a comment
 has been edited or not.

#### Description of Task to be completed?
- Add a new field to the comments table

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_back.git`
- Change into the project directory -` cd travel_tool_back`
- Install project dependencies using `yarn install`
- Run database migrations using `yarn run db:migrate`
- Seed the database using `yarn run db:seed`
- Start the app using make start
- Navigate to [travela](http://travela-staging.andela.com/) in your browser and login
- Open the developer tools, locate applications tab, copy `jwt-token` in the cookies
- Check `.env.example` and set up your `.env` appropriately
- Get back to your command line, Install dependencies and start the server - `make start`
- In`Postman`, use the `jwt-token` to send a request by adding a `Header`, key `Authorization` and value the `token`.
- In Postman, in the `x-www-form-urlencoded` body options;
    - [x] Create a new request
    - [ ] Add a new comment to the request created
- You should be able to add a comment on `localhost:5000/api/v1/comments`
- The response should include the newly added comment with `isEdited: false` 

#### Any background context you want to provide?
This chore is part of the implementation for this [PR](https://github.com/andela/travel_tool_back/pull/48)

Some tests are failing as a result of this [PR](https://github.com/andela/travel_tool_back/pull/46), currently, it is being worked on.

#### What are the relevant pivotal tracker stories?
[#160595712](https://www.pivotaltracker.com/story/show/160595712)

#### Screenshots (if appropriate)


After adding a new comment, `isEdit` is set to default value `false`


<img width="1280" alt="screen shot 2018-09-18 at 22 06 32" src="https://user-images.githubusercontent.com/12033494/45710724-1c57c900-bb90-11e8-8bf5-766c20b3d088.png">


After editing a comment, `isEdit` is set to value of `true`

<img width="1277" alt="screen shot 2018-09-18 at 22 08 20" src="https://user-images.githubusercontent.com/12033494/45710708-106c0700-bb90-11e8-91cd-08fac8230601.png">

#### Questions:
None
